pymc/distributions/multivariate.py

```diff
@@ -343,6 +347,14 @@ def dist(cls, nu, Sigma=None, mu=None, cov=None, tau=None, chol=None, lower=True
         assert_negative_support(nu, "nu", "MvStudentT")
         return super().dist([nu, mu, cov], **kwargs)
 
+    def get_moment(rv, size, nu, mu, cov):
+        moment = mu
+        if not rv_size_is_none(size):
+            if isinstance(size, int):
+                size = (size,)
+            moment = at.full((*size, mu.size), moment)
+        return moment
```

Comment:
The logic here can be simplified by exactly following how `get_moment` method is added for MvNormal distribution in this PR -  https://github.com/pymc-devs/pymc/pull/5171/files.

Reply:
Yes, you are right @Sayam753. Applied this change.